### PR TITLE
user is told whether word is invalid

### LINKED ISSUE #8: As a single player I want to know if my word is valid so that i do not waste a turn
#7 
#43

### PR DESCRIPTION
**Pull request by RyanSelesnik**
closes #8

### CODE REVIEW

**Comment by RyanSelesnik** [comment]
**Acceptance test **
- A user is prompted with a message if the word entered is not valid

File: src/controllers/wordController.js [hunk 1]
```diff
@@ -1,30 +1,39 @@
...
+  const wordOfTheDay = getWordOfTheDay();
+  if (request.body.guess === wordOfTheDay) {
+    response.json('word of the day')
+  } else {
+    response.json('not word of day')
+  }
+};
+
+const wordIsValid = (request, response) => {
+  const x = request.body.guess
```
Comment (raelware) [inline src/controllers/wordController.js:29]: please rename this variable to improve code readability and conform to our style guide.

File: src/__tests__/words.spec.js [hunk 2]
```diff
@@ -15,20 +15,33 @@ const validWords = dictionary.getDictionary();
...
   const wordOfTheDay = wordController.getWordOfTheDay()
-  const body = { guess: wordOfTheDay };
-  const response = await request(app).post('/').send(body)
+  const body = { guess: wordOfTheDay }
+  const response = await request(app).post('/isWordOfTheDay').send(body)
   expect(response.statusCode).toBe(200)
   expect(response.body).toBe('word of the day')
 })
 
+describe('When making a request to /wordIsValid', () => {
```
Comment (raelware) [inline src/__tests__/words.spec.js:32]: Please add a test for the negative case so you can be assured this function is working properly.

**Review by raelware — COMMENTED** [review]
Overall the code is very readable and well structured.

**Review by raelware — APPROVED** [review]
